client/landing/gutenboarding/onboarding-block/acquire-intent/vertical-select/index.tsx

```diff
@@ -63,8 +63,20 @@ const VerticalSelect: React.FunctionComponent = () => {
 	const showArrow = ! siteTitle && ! siteVertical && inputText.length > 2;
 
 	const animatedPlaceholder = useTyper(
-		[ NO__( 'football' ), NO__( 'shopping' ), NO__( 'cars' ), NO__( 'design' ), NO__( 'travel' ) ],
-		isInputEmpty
+		[
+			NO__( 'photography' ),
+			NO__( 'blogging' ),
+			NO__( 'travel' ),
+			NO__( 'marketing' ),
+			NO__( 'fashion' ),
+			NO__( 'shopping' ),
+			NO__( 'design' ),
+			NO__( 'real estate' ),
+			NO__( 'food' ),
+			NO__( 'sports' ),
+		],
+		isInputEmpty,
+		{ delayBetweenWords: 800, delayBetweenCharacters: 110 }
```

Comment:
While this is a perfectly good solution, let's make better use of your designer's taste and make these the default values. You can change the default values [here](https://github.com/Automattic/wp-calypso/blob/master/client/landing/gutenboarding/hooks/use-typer.ts#L29-L32). 

Non-blocker nonetheless.

Reply:
Agreed, this is brand new and we might just set these as the defaults which were arbitrary.